packages/components/src/primitives/Icon/index.tsx

```
@@ -25,10 +25,13 @@ const DEFAULT_SIZE = 24;
 
 const loadIcon = (name: IKeyOfIcons) =>
   new Promise<typeof Svg>((resolve) => {
-    void ICON_CONFIG[name]().then((module: any) => {
+    void ICON_CONFIG[name]?.().then((module: any) => {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      ComponentMaps[name] = module.default as typeof Svg;
-      resolve(ComponentMaps[name]);
+      if (module && module.default) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)